hasher/src/roinput.rs

```diff
@@ -73,7 +73,7 @@ impl ROInput {
     }
 
     /// Append a `Hashable` input
-    pub fn append_hashable(&mut self, input: impl Hashable) -> &mut Self {
+    pub fn append_hashable(&mut self, input: &impl Hashable) -> &mut Self {
```

Comment:
Great PR.  I should have done this.  Can you please also make `append_roinput` take roi as a reference parameter?

Reply:
Or you can leave it.  Up to you.